### ISSUE TITLE
unlocking Specialites Increase is way to big

### ISSUE THREAD

**Issue by Conchipedia** [StrangeLoopGames/EcoIssues#3546]
Version: 0.6.4.1 alpha

![snap1241](https://user-images.githubusercontent.com/29383891/35186211-7f3467ea-fe08-11e7-868c-91e4ce8b472d.png)

Tested by learning a skill scroll  and writing down the next visible SP cost for learning another Specialty, no points in Tree Column of Specialties affects the SP needed to learn a new Specialty.
To Lonewolf a player needs to have a massive **1,699,265** SP this is JUST to learn the Specialty, i didn't even attempt to work out points in the tree part.

Unlearning and adding points into the sub tree does not change these values, and anyone Worried about Lonewolfing by Giving the Player a FREE Stone Axe, Wooden Shovel, Stone Pickaxe and a Workbench need NOT be concerned, at 100sp a day that will take **16,992** days (roughly **46 years**) to get just the Specialties.

**Comment by Huntermeign** [comment]
This does seem ridiculous. Maybe cap it at some point. Otherwise to many servers will just find a way around this if they have a small player base.

**Comment by midn8t** [comment]
well in tool set its option to change it along with skillrate and skillbase . our server found that best play rate is baseskill 1 SKillspeedrate 6 and .1 for skill specialty cost which by time u unlock the fourth specially makes it = 38  we found that its more enjoyable to earn skill points faster then to earn earning more per day for example we found that earning 8 pre day but rate was high enough where day = in game day and in game day 24 hour took = hour 1 hour  so  in one hour of real play time u earn 8 skill points. we then on our server tweeked how long it takes for world to progress through technology by changing the skill cost of skill specialist cost from .3 skill mutiplayer is a compounding thing  so .1 = 10 percent   in cress so on our server we run a 10% skill cost in cress for each Specialty skill.

I do think default should maybe be changed but as of now we love fact that the server admins runners of the dedicated servers have been giving so many options to change values how our community want them to be.  - I gave info above so that developers could see how me and my community have for values running. - we are on the Wisconsin Public Server.  we always believed that when your playing you should be rewarded for doing things instead of just logging off and waiting so that is why we made change to skillrate and pre day = so you feel like your being rewarded as you play.

**Comment by midn8t** [comment]
our server have been using following setup skillrateprogress speed 6 Skillratebase 1 Specialtyskillincress .1 for a compound 10 percent incress instead of 1 which is = to 100% - dedcaited tool set for admins gives us the abilty to tailor these settings for our own community base. as of right now if you look at the server list most of the dedicated servers are for small groups of people majority of them are 4 or 5 players active. and all of them have low or very low settings for the Specialty skill in cress - the highest level they have is 16 i believe which would be equal to 160% compounding...

my own community prefer the lower rating - with faster speed progress with lower daily out come so they get rewarded while there paying faster but gaining less preday  - the Specialty Skill we thought that 10% componding seem be about right by time you unlock 2nd Specialty skills it then equal about 22 points next one pushes it to 24 something and  fourth one hit comes to 26  so with skillbase set to 1 most players earning with a balance diet 18 day

**Comment by MarjieAVolk** [comment]
Yeah the steep curve was basically intended to cap the number of specialties you can take, the higher ones weren't supposed to be reachable.  So the numbers are kind of silly.

We did some math and some discussion, and came up with these numbers to use instead:
0, 5, 20, 100, 250, 500, 800, 800, 800 [... 800 for everything else]

**Comment by MarjieAVolk** [comment]
@midn8t I think you are right that skill points can't be the only thing driving people to cooperate.  On the other hand, skill points are one of the primary challenges in the game, that drive a lot of the central mechanics like food and housing, so the amount that things cost needs to be at a level that is challenging.  But I agree that we also need to encourage interaction in other ways.  And that we do have a problem with people feeling stuck and logging out to wait for better stuff.  It's kind of a central problem to Eco, since the game is designed to be able to do alright without having to log too many hours.  So with that as a central tenet, it's hard to prevent diminishing returns on time spent in game.

**Comment by midn8t** [comment]
I have idea that may or may not help just thought, 

how about two diff ways to earn skill points one for specialties that are done with research and other with doing stuff that being skills inside the skill tree.

separating the two to make skill points earn t over time being used for advancing the world and skill points earn t by doing those specializes for in cress your ability to do the job.

so when people are in game playing they get rewarding with in cress in that skill set in that they are using current as they play and then time management the current system for advancing the technology over course of time.  ?

**Comment by MarjieAVolk** [comment]
Hmmmm that is a cool idea
